initdb.py

```diff
@@ -165,10 +161,15 @@ def main(username='', password=''):
 
     # Close cursor
     mysql_cursor.close()
+    
     # Close connection
     mysql_connection.close()
 
     # Print completed
     print("Database set up completed.")
```

Comment:
This is probably not needed anymore. We can assume OK upon completion without error.